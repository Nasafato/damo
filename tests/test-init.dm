num x = 5.0; 
string y = "hello"; 
bool a = true;
print_num(x+3.0);
//print_string(y);
print_bool(a);



