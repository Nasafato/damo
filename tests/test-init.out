8.000000
hello
1
